Sources/RichTextKit/Actions/RichTextAction+Button.swift

```diff
@@ -47,6 +53,7 @@ public extension RichTextAction {
                 action.icon
                     .frame(maxHeight: fillVertically ? .infinity : nil)
                     .contentShape(Rectangle())
+                    .foregroundColor(systemSpecificTextColor)
```

Comment:
My idea was that you should be able to add a `foregroundColor` to the entire side panel / format sheet and have all the buttons adapt automatically. :)

Reply:
Fixed